returnn/datasets/lm.py

```diff
@@ -1177,6 +1184,7 @@ def __init__(
         add_extra_begin_lemma: float = 1.0,
         extra_end_lemma: Optional[Dict[str, Any]] = None,
         add_extra_end_lemma: float = 1.0,
+        lexicon_strategy: LexiconStrategy = LexiconStrategy.PICK_RANDOM,
```

Comment:
Avoid the enum, just use a `str` (makes the Sis config much simpler, also the code here simpler, also more consistent to our other code).